backend/Search.hs

```diff
@@ -637,6 +638,7 @@ renderFile userfilepath s = H.docTypeHtml $ do
     cssStyle =
       "body{background:#fdf6e3}\
       \body,pre{margin:0}\
+      \pre{overflow-x:auto}\
```

Comment:
Does this mean we’ll get a scrollbar under each long line of code? Isn’t this going to be terrible on narrow screens? I assumed mobile users could just drag the viewport around (no scrollbars at all).

Reply:
No, there is only one `<pre>` element that surrounds the entire table (whose rows are source code lines).

I think that the situation when the user has to drag the viewport is considered to be bad, since it means that the website is not optimised for the user’s device. My theory is that the idea is that you are supposed to fit your content in the user’s viewport and control overflow yourself, e.g. by explicitly requesting scrollbars where necessary.

Reply:
Okay then. (I don’t remember how exactly I structured it and why). If that’s just the outer `<pre>`, then I’m fine with this.